build-extras.gradle

```diff
@@ -0,0 +1,106 @@
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+ext {
+  commonsLang3Version = "3.8.1"
+  forgerockVersion = "5.5.1"
+  retrofit2Version = "2.5.0"
+  jacksonVersion = "2.9.7"
+  okHttpVersion = "3.12.0"
+}
+
+repositories {
+  google()
+  mavenCentral()
+  jcenter()
+  mavenLocal()
+
+  maven { 
+    credentials {
+      username = ARTIFACTORY_USERNAME
+      password = ARTIFACTORY_PASSWORD
+    }
+    url "https://trusona.jfrog.io/trusona/sdk-releases" 
```

Comment:
Do we need to provide artifactory credentials here? I think we just need them in the publish section.

Reply:
yes we do - to access the legacy SDK